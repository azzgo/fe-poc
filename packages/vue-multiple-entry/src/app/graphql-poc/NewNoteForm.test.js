import { shallowMount } from '@vue/test-utils'
import NewNewForm from './NewNoteForm.vue'

test('simple render', () => {
  const comp = shallowMount(NewNewForm)
  expect(comp.html()).toMatchSnapshot()
})
