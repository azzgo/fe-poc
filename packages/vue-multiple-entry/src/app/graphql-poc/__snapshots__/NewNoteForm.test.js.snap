// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple render 1`] = `
<div class="note-creator shadow-2">
  <form class="row">
    <input type="text" placeholder="Note Title" class="col-xs-10 title">
    <input type="text" placeholder="Note Content" class="col-xs-10">
    <div class="actions col-xs-12 row between-xs">
      <button type="submit" class="btn-light">Done</button>
    </div>
  </form>
</div>
`;
